backend/scripts/port-manager.js

```
@@ -0,0 +1,117 @@
+
+switch (command) {
+  case 'kill':
+    killPort(port);
+    break;
+  case 'check':
+  case 'list':
+    listPortProcesses(port);
+    break;
+  case 'help':
+  default:
+    console.log(`
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)